### PR TITLE
Release 4.12.0

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 4.12.0, needed for https://github.com/PrestaShop/PrestaShop/issues/22668
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Please perform a standard testing campaign for upgrade module release. Only significant change should be https://github.com/PrestaShop/autoupgrade/pull/376

### CODE REVIEW

**Comment by matks** [comment]
@PierreRambaud Should we add `index.php` files in all `.github` folders to "protect" them?

File: Readme.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # 1-Click Upgrade
 
 [![Build Status](https://travis-ci.com/PrestaShop/autoupgrade.svg?branch=dev)](https://travis-ci.com/PrestaShop/autoupgrade)
```
Comment (Progi1984) [inline Readme.md:3]: Travis has been removed

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: Readme.md (hunk 1 above)
Comment (matks) [reply Readme.md:3]: Fixed in https://github.com/PrestaShop/autoupgrade/pull/382/

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @matks !

- Simple upgrade from 1.7.6.9 to 1.7.7.x : OK
- Upgrade from 1.7.5.2 to 1.7.7.x, with rollback : OK
- Simple upgrade from 1.7.4.2 to 1.7.7.x : OK
- Upgrade from 1.7.3.3 to 1.7.7.x, with rollback : OK
- Simple upgrade from 1.7.2.4 to 1.7.7.x : OK
- Upgrade from 1.7.1.2 to 1.7.7.x, with rollback : OK
- Simple upgrade from 1.7.0.6 to 1.7.7.x : OK
- Upgrade from 1.6.1.24 to 1.7.7.x : KO (as expected) but rollback is OK.


So it's OK ! QA :heavy_check_mark:

**Comment by Progi1984** [comment]
Thanks @matks & @Robin-Fischer-PS

**Comment by matks** [comment]
@Robin-Fischer-PS Thank you so much for the amazing QA work!

**Pull request merged**